tests/io-tests/io.test.js

```diff
@@ -0,0 +1,89 @@
+const glob = require('glob');
+const fs = require('fs');
+const cp = require('child_process');
+
+const COMPILER_TIMEOUT = 60000; // ms, for each compiler run (including startup)
```

Comment:
Really had to bump these - took a while and was hitting timeouts. Might be worth investigating?
<img width="824" alt="Screenshot 2023-07-03 at 3 26 54 PM" src="https://github.com/brownplt/pyret-lang/assets/16689974/fc8f9f8b-580b-400f-a220-81d42f58d048">